### PR TITLE
make build reproducible

### PR DESCRIPTION
**Pull request by hboutemy**
## Description
Make your build reproducible: see https://maven.apache.org/guides/mini/guide-reproducible-builds.html

On your next public release, I'll add it to Reproducible Central list https://github.com/jvm-repo-rebuild/reproducible-central

## Type of change
Updated plugins versions to get Reproducible Builds-aware versions
Added timestamp to enable Reproducible feature, that maven-release-plugin will update during the automated release process

## Checklist
 - [x] I have read the [contributing guidelines](https://www.eclipse.org/jkube/contributing)
 - [x] I signed-off my commit with a user that has signed the [Eclipse Contributor Agreement](https://www.eclipse.org/legal/ECA.php)
 - [ ] I Added [CHANGELOG](../CHANGELOG.md) entry
 - [ ] I have implemented unit tests to cover my changes
 - [ ] I have updated the [documentation](../kubernetes-maven-plugin/doc) accordingly
 - [x] No new bugs, code smells, etc. in [SonarCloud](https://sonarcloud.io/dashboard?id=jkubeio_jkube) report
 - [ ] I tested my code in Kubernetes
 - [ ] I tested my code in OpenShift

### CODE REVIEW

**Comment by manusa** [comment]
- [ ] https://dev.eclipse.org/ipzilla/show_bug.cgi?id=22780
- [ ] https://dev.eclipse.org/ipzilla/show_bug.cgi?id=22781

File: pom.xml [hunk 1]
```diff
@@ -134,11 +135,21 @@
             </excludes>
           </configuration>
         </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-jar-plugin</artifactId>
+          <version>3.2.0</version>
```
Comment (rohanKanojia) [inline pom.xml:141]: Shall we move this version to a property too?

Comment (manusa) [reply pom.xml:141]: Yep, good candidate for a help-wanted issue.

Comment (manusa) [reply pom.xml:141]: OK, for some reason I thought these plugins were already there (I didn't even create the CQs for these).
We definitely need to set these as properties form the start.

Comment (hboutemy) [reply pom.xml]: ok, properties defined in da100d7

**Comment by rohanKanojia** [comment]
@hboutemy: Hi, ECA check seems to be failing. Maybe you forgot to add "Signed off By: ..." footer in your commit message

**Review by manusa — APPROVED** [review]
LGTM, thx for contributing!

**Comment by hboutemy** [comment]
build confirmed reproducible for jkube 1.1.0 release: https://github.com/jvm-repo-rebuild/reproducible-central#org.eclipse.jkube:jkube

**Comment by manusa** [comment]
Good news, thank you very much for this Hervé.

**Pull request merged**